src/main/kotlin/exercise1/task5/ReverseInteger.kt

```diff
@@ -27,7 +27,16 @@ package org.jetbrains.exercise1.task5
  */
 
 internal fun reverseInteger(x: Int): Int {
-    TODO("Implement me!!!")
+    if(x !in -1_000_000..1_000_000) throw IllegalArgumentException("x in the invalid range!")
+    var res = 1
```

Comment:
Ovde možemo izbeći mutabilne promenljive na sledeći način:
```suggestion
    val number = if(x >= 0) 1 else -1
    return abs(number).toString().reversed().toInt() * number
```